mpi-native/examples/mpi_allgather.cpp

```diff
@@ -3,43 +3,14 @@
 
 #include <faabric/mpi-native/MpiExecutor.h>
 #include <faabric/util/logging.h>
+
 int main(int argc, char** argv)
 {
-    auto logger = faabric::util::getLogger();
```

Comment:
This chunk of code seemed to be duplicated over all MPI native examples so I factored it out